unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -739,4 +746,21 @@ private static void ShowGenericNotification(string message)
         notificationModel.type = NotificationFactory.Type.GENERIC;
         HUDController.i.notificationHud.ShowNotification(notificationModel);
     }
+
+    private void SetObjectCullingStarting(bool isStart)
+    {
+        if (setObjectCullingStartingCoroutine != null)
+            CoroutineStarter.Stop(setObjectCullingStartingCoroutine);
+
+        if (isStart)
+            CoroutineStarter.Start(StartObjectCullingWithDelay(CULLING_ACTIVATION_DELAY));
+        else
+            Environment.i.platform.cullingController.Stop();
+    }
+
+    private IEnumerator StartObjectCullingWithDelay(float delay)
+    {
+        yield return new WaitForSeconds(delay);
+        Environment.i.platform.cullingController.Start();
+    }
```

Comment:
I think I already tried on the #663 and I had the same problem. anyway let me confirm and I will tell you 👍

Reply:
Tested with the #663 and I still have the same problem. I have also tested different approach to avoid this delay hack but nothing that fix it for now. If you don't mind @BrianAmadori, could we have a quick meet to review the culling flow? You may see something I don't.

Reply:
After the pair programming with @BrianAmadori, we have finally solved the problem! Changes done in https://github.com/decentraland/unity-renderer/pull/650/commits/1b7bce84d84edf87423283d8f6c12f4a5bbc8473 and https://github.com/decentraland/unity-renderer/pull/650/commits/19c0c67fbe32f3345b4ba06b0f47ccc5f0f6236b. Thanks Brian!